source/stylesheets/_refills-header.scss

```diff
@@ -9,13 +9,13 @@
 
   @include media($refills-medium-screen) {
     margin-bottom: 4em;
-    padding: 6em 0 10em;
+    padding: 6em 0 8em;
   }
 
   @include media($refills-large-screen) {
     border-bottom: 2px solid darken($refills-hero-background, 5%);
     margin-bottom: 4em;
-    padding: 6em 0 11em 0;
+    padding: 6em 0 8em 0;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `6em 0 8em` instead of `6em 0 8em 0`